### PR TITLE
Issue 9

### PR DESCRIPTION
**Pull request by alexandertv**
Implements genericModel and LDAModel with tests.

### CODE REVIEW

**Comment by jpwahle** [comment]
Please fix merge problems before review.

**Comment by jpwahle** [comment]
Please also make sure to merge into dev, not master.

File: nlp_land_prediction_endpoint/middleware/auth.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 from datetime import datetime, timedelta
 from typing import Optional
 
-import jwt
+import jwt  # type: ignore
```
Comment (jpwahle) [inline nlp_land_prediction_endpoint/middleware/auth.py]: Why do we need to ignore the types of jwt now?

File: .flake8 [hunk 2]
```diff
@@ -2,6 +2,5 @@
 exclude = .venv, venv/
 docstring-convention = google
 max-line-length = 100
-
-ignore = D104, D205, D415, W503
+ignore = D104, D205, D415, ANN101, W503
```
Comment (jpwahle) [inline .flake8]: Why are we excluding self annotations? We can just define them for a class and then use the generic type to annotate self.

Comment (jpwahle) [reply .flake8]: Please include them again and use the annotations so in the IDEs we get method suggestions etc.

Comment (jpwahle) [reply .flake8]: If you need an example, consider the generic client of the crawler.
https://github.com/ag-gipp/NLPLand/blob/a6f451fdb3d3425df03075e8e3660977e4db502b/nlpland/client/generic/genericclient.py#L10

**Review by jpwahle — CHANGES_REQUESTED** [review]

File: .flake8 (hunk 2 above)
Comment (tom-nslt) [reply .flake8]: I misread this annotation and thought it was soon to be deprecated, like W503 (Binary operators). Now every self is annotated.

File: nlp_land_prediction_endpoint/middleware/auth.py (hunk 1 above)
Comment (tom-nslt) [reply nlp_land_prediction_endpoint/middleware/auth.py]: My mistake, too hastily tried to revert the pipeline "fixes". JWT as well as requests are no longer ignored

**Review by jpwahle — APPROVED** [review]
LGTM

**Pull request merged**